include/jwt-cpp/jwt.h

```diff
@@ -3590,8 +3610,8 @@ namespace jwt {
 	 * Return a builder instance to create a new token
 	 */
 	template<typename json_traits>
-	builder<json_traits> create() {
-		return builder<json_traits>();
+	builder<default_clock, json_traits> create(default_clock c = {}) {
```

Comment:
there also should be a create method that allows users to specify both types 

https://github.com/Thalhammer/jwt-cpp/blob/953aab6fe47be9e92fcbf5fb8418dee0d682e7ca/include/jwt-cpp/jwt.h#L3765

Reply:
Done